modules/volume.py

```diff
@@ -143,7 +144,15 @@
 # Starts Volume profiling
   - volume: action=profile
              volume="{{ volname }}"
-             profile_state="start"         
+             profile_state="start"   
+    run_once: true
+    
+# Sets Volume options
```

Comment:
Volume set can be worked separately. You can keep your current changes limited to volume profile.